jax/_src/scipy/signal.py

```diff
@@ -284,6 +284,37 @@ def convolve2d(in1: Array, in2: Array, mode: str = 'full', boundary: str = 'fill
     - :func:`jax.numpy.convolve`: 1D convolution
     - :func:`jax.scipy.signal.convolve`: ND convolution
     - :func:`jax.scipy.signal.correlate`: ND correlation
+
+  Examples:
+    A few 2D convolution examples:
+
+    >>> x = jnp.arange(1, 10).reshape(3,3)
```

Comment:
I think the examples would be clearer if `x` and `y` were different shapes. Perhaps make `x` a 2x2 array? And perhaps construct it explicitly like `y` so its easier to visualize the structure of the two convolutions.

Reply:
Thanks! Updated the examples with `x` and `y` having different shapes.